src/NewsletterSignup/IbcNewsletterSignup.vue

```diff
@@ -0,0 +1,364 @@
+  line-height: 1.5;
+  height: auto;
+  box-sizing: border-box;
+  transition: all .15s;
+  backdrop-filter: blur(10px);
+  font-family: var(--ds-font-family, sans-serif);
+  font-weight: 400;
+  margin: 0;
+}
+.email__form__input__input:hover {
+  background-color: rgba(255,255,255,0.13);
+}
+.email__form__input__input:focus {
+  box-shadow: inset 0 0 0 1.5px #66A1FF;
+  background-color: rgba(0,0,0,0.2);
+}
+.email__form__input__input::-webkit-input-placeholder,
+.email__form__input__input::placeholder {
+  color: rgba(255, 255, 255, 0.5);
+  transition: color .15s;
+}
+.email__form__input__input:hover:not(:focus)::-webkit-input-placeholder,
+.email__form__input__input:hover:not(:focus)::placeholder {
+  color: rgba(255, 255, 255, 0.8);
+}
+.forwards-enter-active,
+.forwards-leave-active,
+.backwards-enter-active,
+.backwards-leave-active {
+  transition: all .5s ease-in-out;
+  position: absolute;
+}
+.forwards-enter {
+  opacity: 0;
+  transform: translateY(50px);
+}
+.forwards-enter-to {
+  opacity: 1;
+  transform: translateY(0);
+}
+.forwards-leave {
+  opacity: 1;
+  transform: translateY(0);
+}
+.forwards-leave-to {
+  opacity: 0;
+  transform: translateY(-50px);
+}
+.backwards-enter {
+  opacity: 0;
+  transform: translateY(-50px);
+}
+.backwards-enter-to {
+  opacity: 1;
+  transform: translateY(0);
+}
+.backwards-leave {
+  opacity: 1;
+  transform: translateY(0);
+}
+.backwards-leave-to {
+  opacity: 0;
+  transform: translateY(50px);
+}
+.fade-enter-active,
+.fade-leave-active {
+  transition: opacity 5s;
+}
+.fade-enter, .fade-leave-to {
+  opacity: 0;
+}
+.fade-enter-to, .fade-leave {
+  opacity: 1;
+}
+@media screen and (max-width: 800px) {
+  .wrapper {
+    grid-template-columns: 25% 75%;
+  }
+}
+@media screen and (max-width: 600px) {
+  .wrapper {
+    padding-left: 1rem;
+  }
+  .image {
+    display: none;
+  }
+  .text {
+    grid-column: 1/3;
+  }
+}
+</style>
+
+<script>
+import GraphicsPlanes from "./GraphicsPlanes"
+import DsButton from "./DsButton"
+import IconArrowRight from "../Icons/IconArrowRight"
+import IconIbc from "../Icons/IconIbc"
+import querystring from "querystring"
+
+export default {
+  props: {
+    banner: {
+      type: Boolean,
+      default: true
+    },
+    fullscreen: {
+      type: Boolean,
+      default: false
+    }
+  },
+  watch: {
+    fullscreen() {
+      this.setHeight(`step${this.step}`)
+    }
+  },
+  components: {
+    GraphicsPlanes,
+    DsButton,
+    IconArrowRight,
+    IconIbc
+  },
+  data: function() {
+    return {
+      step: 0,
+      transition: "forwards",
+      pageMinHeight: null,
+      email: null,
+      subscriptions: {
+        tools: false,
+        ecosystem: false
+      },
+      url: "https://zcs1.maillist-manage.com/campaigns/weboptin.zc",
+      commonFormData: {
+        "zc_trackCode": "ZCFORMVIEW",
+        "viewFrom": "URL_ACTION",
+        "submitType": "optinCustomView",
+        "emailReportId": "",
+        "zx": "129a50c11",
```

Comment:
is this zoho's newsletter id for ibc? @fadeev

Reply:
No, it is not. You need `lD`, `zcld` and `zc__formIx`. If the subscription link on `cosmos.network/ibc` is correct, then we can fish out these values by inspecting the source of the form.

```js
{
  lD: "16352f88328053f9",
  zcld: "16352f88328053f9",
  zc_formIx: "4ef47fbb86ab6668b78ce576d7aa791940fbf6cd35f5c50c"
}
```

Please, double check.

Reply:
![image](https://user-images.githubusercontent.com/1021101/78388235-7fa02500-7595-11ea-8939-0c64e5399aeb.png)